cas_implementations/src/eav/file/actor.rs

```diff
@@ -167,7 +167,9 @@ impl EavFileStorageActor {
         Ok(entity_attribute_value_inter
             .into_iter()
             .filter(|e| e.is_ok())
-            .map(|eav_content| EntityAttributeValue::from_content(&JsonString::from(eav_content.unwrap())))
+            .map(|eav_content| {
+                EntityAttributeValue::from_content(&JsonString::from(eav_content.unwrap()))
```

Comment:
You could just do `eav_content.unwrap().into()`

Reply:
@sphinxc0re please provide code suggestions in github using the new functionality, or make minor changes directly and push them up